go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/nnev/website
+
+go 1.19
```

Comment:
Auf eris gibt's nur go in Version 1.15.5, da schlägt dann vermutlich fehl?

Reply:
Also: don't worry about it, fürs erste, aber Upgrade Go auf eris trotzdem bei Gelegenheit

Reply:
okay, lol, ich benutze tatsächlich eine API die aus Go 1.20 kommt =D sekunde, ich fixe.

Reply:
Done. Hab auch verifiziert jetzt, dass das mit Go auf eris baut.